src/common/oscapxml.c

```diff
@@ -52,33 +53,35 @@
 #include "source/validate_priv.h"
 #include "source/xslt_priv.h"
 
-#ifndef OSCAP_DEFAULT_SCHEMA_PATH
-const char * const OSCAP_SCHEMA_PATH = "/usr/local/share/openscap/schemas";
-#else
-const char * const OSCAP_SCHEMA_PATH = OSCAP_DEFAULT_SCHEMA_PATH;
-#endif
+const char *const OSCAP_SCHEMA_PATH = OSCAP_DEFAULT_SCHEMA_PATH;
+const char *const OSCAP_XSLT_PATH = OSCAP_DEFAULT_XSLT_PATH;
+const char *const OSCAP_CPE_PATH = OSCAP_DEFAULT_CPE_PATH;
 
-#ifndef OSCAP_DEFAULT_XSLT_PATH
-const char * const OSCAP_XSLT_PATH = "/usr/local/share/openscap/xsl";
-#else
-const char * const OSCAP_XSLT_PATH = OSCAP_DEFAULT_XSLT_PATH;
-#endif
-
-#ifndef OSCAP_DEFAULT_CPE_PATH
-const char * const OSCAP_CPE_PATH = "/usr/local/share/openscap/cpe";
-#else
-const char * const OSCAP_CPE_PATH = OSCAP_DEFAULT_CPE_PATH;
-#endif
-
-/* return default path if pathvar is not defined */
+/*
+ * If environment variable specified by 'pathvar' is defined, returns value of this environment variable.
+ * If this environment variable is not defined, returns default path, provided by 'defpath' argument.
+ */
 static const char * oscap_path_to(const char *pathvar, const char *defpath) {
 	const char *path = NULL;
 
 	if (pathvar != NULL)
 		path = getenv(pathvar);
 
-	if (path == NULL || oscap_streq(path, ""))
+	if (path == NULL || oscap_streq(path, "")) {
```

Comment:
I suggest extracting the body of the `if` statement to a separate function, so we have functions without `ifdef`s in the body. Remember, functions containing only one line [are considered as good practice by some under certain conditions](https://sites.google.com/site/unclebobconsultingllc/one-thing-extract-till-you-drop) and are a non-issue in C performance-wise :-)